eventually-v0-5-0/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "eventually-v0-5-0"
+version = "0.5.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = "1.0.45"
```

Comment:
is `anyhow` used anywhere? shouldn't generally be used in a library anyway

Reply:
Good catch, forgot about that! I was using it when writing some middle-layer errors, but I don't need it anymore 👍🏻 

On a different note, I think it's still ok to use `anyhow` in `enum` errors (with `thiserror`) when the error is supposed to be "opaque". But I guess one could use `Box<dyn std::error::Error>` too in that case 😅

Reply:
when would you have opaque error types?

if you are providing a mechanism for downstream users to use their own error types somehow, i think it would be better for this to be a generic variant, rather than an opaque error. Otherwise you're still leaving the consumers to try and downcast the error types or whatever.

as in,

```rust
pub enum Error<T> {
   Io(io::Error),
   Custom(T),
}
```

Reply:
Yeah that's what I do, but in case of complex errors this gets kinda cumbersome, so at some point using some opaque errors is an option. Not happening in the current code however, and hoping that won't happen with the new PRs 😄